cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -136,10 +137,15 @@ public DebeziumOperandController(KubernetesClient kubernetesClient, DebeziumOper
 
         kcsb.withImage(shardMetadata.getContainerImage());
 
+        final String operatorName = getOperatorName(connector);
+        LOGGER.debug("Assigned {}connector.operator as {} for connector {}", BF2_DOMAIN, operatorName,
+            connector.getMetadata().getName());
+
         final KafkaConnect kc = new KafkaConnectBuilder()
             .withApiVersion(Constants.RESOURCE_GROUP_NAME + "/" + KafkaConnect.CONSUMED_VERSION)
             .withMetadata(new ObjectMetaBuilder()
                 .withName(connector.getMetadata().getName())
+                .addToLabels(BF2_DOMAIN + "connector.operator", operatorName)
```

Comment:
So there is still something that isn't clear to be @valdar and @lburgazzoli.

If I adjust the `kustomization.yaml` files under `operator-debezium/local` and set the `COS_OPERATOR_ID` as it is from the addon to be `"cos-fleetshard-operator-debezium-1.0.1"`, when a connector create request comes into the operator it deems it as not assigned because the operator id in the pod is the `COS_OPERATOR_ID` but the assigned check reports this:
```
2022-03-22 16:40:50,882 DEBUG [org.bf2.cos.fleetshard.operator.connector.ConnectorController] (EventHandler-connector) Connector default/mctr-c8svnsu3bd8ifa234ni0 is not managed by this operator (assigned=cos-fleetshard-operator-debezium, operating=null).
```

So am I still just misunderstanding what needs to be adjusted to test this locally?